packages/admin/cms-admin/src/redirects/RedirectsTable.tsx

```diff
@@ -80,7 +80,7 @@ export function RedirectsTable({ linkBlock, scope }: Props): JSX.Element {
             field: "source",
             headerName: intl.formatMessage({ id: "comet.pages.redirects.redirect.source", defaultMessage: "Source" }),
             sortable: true,
-            flex: 1,
```

Comment:
You can use different values for `flex`: https://mui.com/x/react-data-grid/column-dimensions/#fluid-width

So you could use `flex: 3` here for example